cumulus_library_opioid/vocab/additional_rules_builder.py

```diff
@@ -63,8 +63,8 @@ def prepare_queries(
                     },
                     join_clauses= [
                         's.rxcui2 = r.rxcui',
-                        ('s.rxcui2 NOT IN (SELECT DISTINCT RXCUI FROM '
-                        f'{prefix}{steward}_rxnconso_keywords)')
+                        # ('s.rxcui2 NOT IN (SELECT DISTINCT RXCUI FROM '
+                        # f'{prefix}{steward}_rxnconso_keywords)')
```

Comment:
Was this manual testing cruft or lines that should just be deleted now?

Reply:
testing cruft, will reset.